src/configure

```diff
@@ -184,6 +189,29 @@ do
   --cudatk-dir=*)
     CUDATKDIR=`read_dirname $1`;
     shift ;; #CUDA is used in src/cudamatrix and src/nnet{,bin} only
+  --host=*)
```

Comment:
I use the configure switches to allow my jenkins instance to build for all our supported architectures in one place.  Hand editting the kadli.mk file defeats this (or at least requires that all my builds have sed calls to replace all the tool definitions not just gcc, but ld, ar, and ranlib).

I want to note that you do not have to set the HOST variable, it is used in a way that when building natively, leaving it empty gives the current behavior.